notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb

```diff
@@ -0,0 +1,625 @@
+{
```

Comment:
<p>What's the correct branding? Vertex Model Garden or Vertex AI Model Garden?</p><p> </p><p>Move H1 heading to first line in next links cell.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Vertex AI Model Garden is the correct one, updated.
This notebook is created in Vertex AI workbench, where we will point our users to launch the notebook. So all cells and heading positions are arranged by the workbench notebook metadata.